### DISCUSSION TITLE
Documentation Work in progress

### DISCUSSION THREAD

**Discussion by appenzellois** [marian-m12l/studio#215] [Q&A]
Hi,
I'm working on the Documentation.

Does anybody know what those 2 settings are about ?

- Opt out of announces
- Allow enriched binary format (includes metadata)

And does anybody have an idean on how to add images to the Wiki?
Except from starting a new discussion and pasting them in the thread to retrieve them by URL afterwards in the Wiki, I have no better idea...

**Comment by marian-m12l** [comment]
The wiki is its own git repository, which you can clone and add images to. This might be outdated, but it seems to be the way: http://mikehadlow.blogspot.com/2014/03/how-to-add-images-to-github-wiki.html

**Reply by appenzellois** [reply]
I found another workaround : start a new issue, add images, grab URL and use it to add images to Wiki, then close issue withour saving it.
The uploaded images remain on the server, even without the issue actually created...

**Reply by marian-m12l** [reply]
@appenzellois Wow I just came upon the documentation, the images make for a massive improvement! I've been willing to add visuals for a long time, thanks!

**Reply by appenzellois** [reply]
^^
I'm not a very skilled programmer, but I'm a Software teacher and Consultant.
This is my small contribution to your extraordinary work!

Now I need some time to write detailled explanations, but I tried to make the images quite self explanatory.
I hope this works. I'd love to have some reader feedback.

Maybe you will notice less user questions, which would be an encouraging sign.

On my to do list I also wrote "make a youtube demo or something alike" :-)

**Reply by appenzellois** [reply]
And then translate it to French (and German if I get motivated enough and have enough time)

**Reply by marian-m12l** [reply]
@appenzellois This is an awesome contribution. I'm biased but I find the images understandable on their own. I added a link to this wiki page in the readme. Thanks!

**Comment by marian-m12l** [comment]
> Does anybody know what those 2 settings are about ?
> 
>     * Opt out of announces

When opening the web UI, an announce may be displayed, if it is newer than the build time and the last display time. This is a way for me to communicate information to the users, such as an important update or in case a device-bricking bug appeared later on. This setting is an opt-out.

> 
>     * Allow enriched binary format (includes metadata)
> 

The binary/raw format supported by the lunii device does not contain as much metadata as you can have with the editable archive format. With this option, you can add such metadata while remaining compatible with the device. The added metadata are: the pack title and description (potentially truncated), nodes name (potentially truncated), group identifier, type, and position in the graph. This makes it possible to recover more data when converting back from the device into the editor. This works only for "binary / raw / lunii firmware v1" format for now, but could also support the newer "fs / lunii firmware v2" format in the future.
